scripts/bump-version.sh

```diff
@@ -75,17 +81,20 @@ fi
 
 git push origin "${TRAVIS_BRANCH}"
 
-# TODO: Publish tag for test and master branch releases
-
-# TODO: auto changelog
+# Publish tags for dev, test and master branch releases only
+if [[ "$TRAVIS_BRANCH" == 'master' || "$TRAVIS_BRANCH" == 'test' || "$TRAVIS_BRANCH" == 'dev' ]]; then
+    git push origin --tags
+else
+    echo "Will not create new tags for feature branch builds"
+fi
 
 # Publish npm module
 if [ "$TRAVIS_BRANCH" = 'master' ]; then
     npm publish --tag latest --access public
-elif [ "$TRAVIS_BRANCH" = 'test' ]; then
+    elif [ "$TRAVIS_BRANCH" = 'test' ]; then
     npm publish --tag beta --access public
-elif [ "$TRAVIS_BRANCH" = 'dev' ]; then
-    npm publish --tag alpha --access public
+    elif [ "$TRAVIS_BRANCH" = 'dev' ]; then
+    ar    npm publish --tag alpha --access public
```

Comment:
The `ar` seems suspicious here.

Reply:
Good spot!
`ar` is more than suspicious here:- it's malicious 🤣
An epic typo on my part:- my bad.